joomla/joomla-cms
# [4.3] Remove obsolete exclude patterns from ruleset.xml for phpcs

Pull Request for Issue # .

### Summary of Changes

This pull request (PR) removes obsolete exclude patterns from the "ruleset.xml" file which is used by phpcs e.g. in the ci tests with Drone.

Obsolete means the corresponding file or folder has been removed (or moved to another place) with some PR but the exclude pattern has been forgotten about.

In detail following entries are removed due to the mentioned PR:
- Line no. 123: File "plugins/editors/tinymce/tinymce.php" has been moved to another place and been changed with PR #39625 .
- Line no. 124: File "plugins/system/cache/cache.php" has been removed with PR #36042 .
- Line no. 239: File "plugins/content/emailcloak/emailcloak.php" has been moved to another place and been changed with PR #38466 .
- Line no. 243: File "plugins/editors/none/none.php" has been moved to another place and been changed with PR #39639 .

The list of obsolete exclude patterns has been created with help of a small tool (PHP CLI script) which I provide with PR #40278 .

Which PR has caused each change has then been "manually" investigated by me.

### Testing Instructions

Code review: Check that the removed exclude patterns belong to files mentioned in the list in the description above and the corresponding files don't exist on a clean, current 4.3-dev branch.

Verify that the PHPCS step succeeded in the Drone ci checks for this PR here.

### Actual result BEFORE applying this Pull Request

File "ruleset.xml" in the Joomla root contains exclude patterns for files which don't exist anymore in the 4.3-dev branch.

### Expected result AFTER applying this Pull Request

File "ruleset.xml" in the Joomla root doesn't contain these exclude patterns.

PHPCS still succeeds in Drone for this PR.

### Link to documentations
Please select:
